### PR TITLE
Return correct message for invalid whatsapp otp

### PR DESCRIPTION
**Pull request by tung2744**
ref #3170

### CODE REVIEW

File: resources/authgear/templates/en/web/__error.html [hunk 1]
```diff
@@ -131,13 +131,18 @@
...
-    </li>
+    {{ end }} {{ end }}
+    {{ else if eq .Error.reason "UserNotFound" }}
+    {{ template "__error_account" . }}
+    {{ else if eq .Error.reason "InvalidCredentials" }}
+      {{ if eq $info.Channel "whatsapp" }}
+        <li>{{ template "error-invalid-whatsapp-otp" }}</li>
+      {{ else }}
+        <li>
+          {{ template "error-invalid-credentials" (dict "AuthenticationType"
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/web/__error.html:142]: I suggest we simply remove the words "SMS" and "Email" to make the error message more generic. I think having the words "Whatsapp", "SMS", "Email" does not make the error message informative.

File: resources/authgear/templates/en/web/__error.html [hunk 2]
```diff
@@ -169,18 +174,23 @@
...
-        {{ template "error-verification-code-invalid-click-to-resend" }}
-      </a>
-    </li>
+    {{ end }}
+    {{ else if eq .Error.reason "InvalidVerificationCode" }}
+      {{ if eq $info.Channel "whatsapp" }}
+        <li>{{ template "error-invalid-whatsapp-otp" }}</li>
+      {{ else }}
+        <li>
+          {{ template "error-verification-code-invalid" }}
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/web/__error.html:183]: I think we should keep using this error message, as it provides a way to resend.

**Review by louischan-oursky — COMMENTED** [review]
@fungc-io What is your thought?

**Comment by tung2744** [comment]
Updated to use generic message

**Pull request merged**